standard/delegates.md

```diff
@@ -207,6 +207,43 @@ Once instantiated, a delegate instance always refers to the same invocation list
 
 > *Note*: Remember, when two delegates are combined, or one is removed from another, a new delegate results with its own invocation list; the invocation lists of the delegates combined or removed remain unchanged. *end note*
 
+> *Note*: Delegate combination and removal is known to fail with an exception in some implementations when the delegate types involved at runtime are different due to variant conversion ([§18.2.3.3](interfaces.md#18233-variance-conversion). The following example demonstrates a situation which may fail.
```

Comment:
Note: to save time, you can just place "§18.2.3.3" in the text, and the automatic section renumbering tool will format the link to the correct anchor.  These are correct, so no need to make any edits.

Reply:
Ah right. :)